packages/lib/src/index.ts

```diff
@@ -36,6 +36,7 @@ export { default as HeatmapMesh } from './vis/heatmap/HeatmapMesh';
 export { default as DataCurve } from './vis/line/DataCurve';
 export { default as Html } from './vis/shared/Html';
 export { default as Annotation } from './vis/shared/Annotation';
+export { default as SelectionMesh } from './vis/shared/SelectionMesh';
```

Comment:
I made `SelectionMesh` part of `lib` since it is quite close to `TooltipMesh` and `PanZoomMesh`. I have to export it to reuse it in the storybook.